.github/workflows/GH-R-CMD-check.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
           try(pak::pkg_install("tidymodels/stacks"))
           try(pak::pkg_install("tidymodels/themis"))
           try(pak::pkg_install("tidymodels/tidymodels"))
-          try(pak::pkg_install("tidymodels/tune"))
+          try(pak::pkg_install("tidymodels/tune#798"))
```

Comment:
TODO before merging this PR: point back to `main`

Reply:
798 is merged!

Reply:
yep, I've updated it to use tune#804 until we're done with that!